plugin-customcommand/lxqtcustomcommand.cpp

```diff
@@ -191,22 +191,26 @@ void LXQtCustomCommand::handleFinished(int exitCode, QProcess::ExitStatus /*exit
     if (exitCode != 0) 
         mOutput = tr("Error");
     
-    updateButton();
     if (mRepeat)
         mTimer->start();
 }
 
 void LXQtCustomCommand::handleOutput()
 {
-    QByteArray data = mProcess->readAllStandardOutput();
-    if(mOutputImage) {
-        mOutputByteArray = data;
-    } else {
-        mOutput = QString::fromUtf8(data);
-        if (mOutput.endsWith(QStringLiteral("\n")))
-            mOutput.chop(1);
+    bool something_read = false;
+    while (mProcess->canReadLine()) {
+        mOutputByteArray = mProcess->readLine();
```

Comment:
should this be
`mOutputByteArray += mProcess->readLine();`
to allow multiline output? like a svg, or just 2 lines of text

Reply:
maybe restricting to 1 line would make it easier/less prone to errors if we were to do something like this:
https://github.com/lxqt/lxqt-panel/issues/2304#issuecomment-3335605892

Reply:
I think 2 lines will be a hassle.

Reply:
alright then lets keep this as suggested by palinek.